File: av1/fibonacci-java/Fibonacci/Fibonacci.java
public class Fibonacci {
    public static int fibonacci(int n) {
        if (n <= 1) {
            return n;
        } else {
            return fibonacci(n - 1) + fibonacci(n - 2);
        }
    }

    public static void main(String[] args) {
        int numeroDeTermos = 10;
        for (int i = 0; i < numeroDeTermos; i++) {
            System.out.println(fibonacci(i));
        }
    }
}

File: av1/fibonacci-js/fibonacci.js
function fibonacci(n) {
    if (n <= 1) {
        return n;
    } else {
        return fibonacci(n - 1) + fibonacci(n - 2);
    }
}

const numeroDeTermos = 10;
for (let i = 0; i < numeroDeTermos; i++) {
    console.log(fibonacci(i));
}
  